Capri.Web/Configuration/SystemSettingsGetter.cs

```diff
@@ -0,0 +1,35 @@
+using Microsoft.Extensions.Options;
+using Capri.Services;
+using Capri.Services.SystemSettings;
+
+namespace Capri.Web.Configuration
+{
+    public class SystemSettingsGetter : ISystemSettingsGetter
```

Comment:
Service should stay in services project. Only `OnChange` watcher should be moved.